pkg/manager/client/discovery.go

```diff
@@ -31,40 +30,35 @@ func NewDiscoveryClient(ctx context.Context, config *rest.Config) (*DiscoveryCli
 	}, nil
 }
 
-func toObj(b []byte, groupVersion string, kind string) interface{} {
-	re, err := regexp.Compile(`""`)
-	if err != nil {
-		logrus.Errorf("Unable to create regex.")
-		return nil
-	}
+func toObj(b []byte, namespace, groupVersion, kind string) interface{} {
 
-	replacement := `"null"`
-	afterReplace := re.ReplaceAllLiteralString(string(b), replacement)
+	replaceString := strings.ReplaceAll(string(b), `"creationTimestamp":null`, `"creationTimestamp":"null"`)
```

Comment:
Hmm? How can `creationTimestamp` is null for an existing object? And why is it treated as a special case?

Reply:
The special case is due to the fact that json requires using null (without double quote) to represent a null field, but yaml needs a string with proper quotes. Let me dig into why the returned timestamp is a null field. @yasker

Reply:
I meant, `null vs "null"` should apply to all fields, not only `creationTimestamp`.

Reply:
@yasker  absolutely make much sense. Just update it to match all invalid null in the format with regexp.